iron-iconset-svg.html

```diff
@@ -69,6 +69,10 @@
       }
 
     },
+    
+    attached: function() {
+      this.style.display = 'none';
```

Comment:
Not in a repeatable CI setting, no. We have gotten good mileage out of the Chrome timeline profiler though.